modules/cluster-logging-visualizer-kibana.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * logging/viewing/cluster-logging-visualizer.adoc
+
+[id="cluster-logging-visualizer-kibana_{context}"]
+= Viewing cluster logs in Kibana
+
+You can view cluster logs in Kibana. There are multiple ways to view and visualize your data that are beyond the scope of this documentation. For more information, refer to the link:https://www.elastic.co/guide/en/kibana/6.8/tutorial-sample-discover.html[Kibana documentation].
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+* Kibana index patterns must exist.
+
+* To list the *infra* and *audit* indices in Kibana, a user must have the `cluster-admin` role, the `cluster-reader` role, or both roles. The default `kubeadmin` user does not have proper permissions to list these indices. 
++
+If you can view the Pods and logs in the `default` project, you should be able to access the these indices. You can use the following command to check if the current user has proper permissions:
```

Comment:
if you can view all projects of the default, kube-* and openshift-*, you should be able to access these indices.